mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -251,28 +238,34 @@ object CaseKeywordCompletion:
       Nil,
       "",
     )
-    val members = result.result()
+    // When in `List(foo).map{cas@@} we don't want to show basic match completion
+    val (includeBasicMatch, prefix) = selector match
```

Comment:
So, 

- (1) `selector` will be `EmptyTree` if it's for pattern match in the PartialFunction,
- (2) in that case, we won't provide basic compeltions, and don't add `match` in the label.

it took some time to understand (1) because it's not obvious to me that selector will be `EmptyTree` if it's for a partial function because the thing we try to complete against (e.g. `List(foo).map`) is an `Apply`, not `Select`.

--- 

what do you think about making `matchContribute` to receive `tree: Select | Apply` (instead of `selector: Tree`, it would be ideal if there's a better name than `tree`) and doing like


```scala
tree match
  case (_: Apply) => (Nil, "")
  case (_: Select) => (List(basicmatch), "match")
```

?